framework/source/class/qx/ui/list/List.js

```diff
@@ -678,10 +678,23 @@ qx.Class.define("qx.ui.list.List",
 
 
     /**
-     * Helper method to update row heights.
+     * Helper method to update group row heights.
      */
     __updateGroupRowHeight : function()
     {
+      /*
+       * In case of having variableItemHeigth set to true,
+       * the group item height has a variable height as well
+       * and will be set again in method _setRowItemSize 
+       * which is a deferred call, being run after all changes.
+       * Resetting the complete item sizes here and setting
+       * the height of the group items, only leads to an
+       * unnecessary flicker of the list items by shrinking and
+       * growing them again.
+       */ 
+      if(this.isVariableItemHeight()) {
```

Comment:
For code consistency with qooxdoo style, there should be a space after `if`. Otherwise, this PR looks good.

Reply:
done.